src/ceph_fuse.cc

```diff
@@ -94,7 +94,8 @@ int main(int argc, const char **argv, const char *envp[]) {
 
   for (auto i = args.begin(); i != args.end();) {
     if (ceph_argparse_double_dash(args, i)) {
-      break;
+      cerr << "-- not allowed with ceph-fuse" << std::endl;
+      exit(0);
```

Comment:
I'm not really happy with disabling it. Should `--` just mean that the next parameter is the mount point?

Reply:
@batrick 

I see multiple issues with this part of the code. 

First of all, the options for loop should be above `global_init()` because the `global_init()` [processes `-m` and removes it from the `args`](https://github.com/ceph/ceph/blob/master/src/common/config.cc#L597) in a normal execution (mounting) of the command. So it is better process the options before `global_init()` and then pass the resulting `args` to `global_init()` for further processing (-m). I need to check more about the `md_config_t::parse_argv()` implementation.

Second one is the `--` meaning issue.